iterators.tex

```diff
@@ -408,7 +409,7 @@
 \begin{itemize}
 \item \tcode{++i} is valid if and only if \tcode{i++} is valid.
 \item If \tcode{i} is incrementable, then both \tcode{++i}
-  and \tcode{i++} advance \tcode{i} to the next element.
+  and \tcode{i++} advance \tcode{i} to denote the next (possibly past-the-end) element.
```

Comment:
I'll reiterate my position that I despise this meaningless requirement and think it should be struck. What's an "element"? What does "denote" mean here? What's a "next element"? What is a "past-the-end" element? These terms have meaning in the context of iterators, but this concept - despite being (mis?) located in iterator.requirements - is not an iterator concept.

At the very least, we should relocate this requirement to `Iterator`.